Gemfile.lock

```diff
@@ -437,4 +441,4 @@ DEPENDENCIES
   whenever (= 0.9.4)
 
 BUNDLED WITH
-   1.13.1
+   1.13.5
```

Comment:
I will put it back anyway, no need to change that version together with this change. Thanks for looking!